src/node_alive/__init__.py

```diff
@@ -0,0 +1 @@
+from autostarter import AutoStarter
```

Comment:
"No newline at end of file"

Reply:
added line updated in a1ef39d19ac8416def3517506f5d571fcb9e1645